src/main/java/com/hellosign/sdk/HelloSignClient.java

```diff
@@ -747,6 +747,27 @@ public File getFiles(String requestId, String format) throws HelloSignException
     }
 
     /**
+     * Retrieves a URL for a file associated with a signature request.
+     *
+     * @param requestId String signature request ID
+     * @return {@link FileUrlResponse}
+     * @throws HelloSignException thrown if there's a problem processing the
+     *         HTTP request or the JSON response.
+     * @see <a href="https://app.hellosign.com/api/reference#get_files">https://app.hellosign.com/api/reference#get_files</a>
+     */
+    public FileUrlResponse getFilesUrl(String requestId) throws HelloSignException {
+        String url = BASE_URI + SIGNATURE_REQUEST_FILES_URI + "/" + requestId;
+
+        HttpClient httpClient = this.httpClient.withAuth(auth).withGetParam(PARAM_GET_URL, "1").get(url);
```

Comment:
@rscarvalho @cmpaul One thing that this does not do is handle the case of a 409 response as per the docs for documents that might still be processing

I saw that the library is still at 1.7, if this was 1.8, `Optional` would be a great to use for things like 404s instead of throwing an exception.